### PR TITLE
feat(frontend): qr code logo and a11y

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

The designers want to display the token logo over the QR code (see PR #3023 for screenshots). Therefore we need to improve the related component to support this option.

At the same time, I noticed that the code lacked of accessibility hint and I removed the border style around it to closer match the Figma design.

# Changes

- Add new property `addressToken` which is used to render a11y hints and logo
- Use `Logo` cmp to render the logo
- Review style

### CODE REVIEW

File: src/frontend/src/lib/components/receive/ReceiveQRCode.svelte [hunk 1]
```diff
@@ -15,19 +27,36 @@
...
+		<article
+			aria-label={replacePlaceholders($i18n.wallet.alt.qrcode_address, {
+				$token: symbol ?? ''
+			})}
+		>
+			<QRCode value={address}>
+				<svelte:fragment slot="logo">
+					{#if nonNullish(icon)}
+						<div class="flex items-center justify-center rounded-lg bg-white p-2">
+							<Logo
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/receive/ReceiveQRCode.svelte]: Nice! That works too sure.

Done in https://github.com/dfinity/oisy-wallet/pull/3024/commits/4e9d2426bb1db5c1d460eb08121fc46ea318c355

File: src/frontend/src/lib/components/receive/ReceiveQRCode.svelte [hunk 2]
```diff
@@ -15,19 +27,36 @@
...
 
 <style lang="scss">
-	div {
-		border: 2px solid var(--color-secondary);
+	.qr-container {
 		max-width: var(--qrcode-max-width, 300px);
 		margin: 0 auto;
 		height: var(--qrcode-height);
-		border-radius: 10px;
-		background: white;
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/receive/ReceiveQRCode.svelte:31]: Indeed. I can add it back if you want?

[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/receive/ReceiveQRCode.svelte:31]: nono, all good! just confirming! thank you!

File: src/frontend/src/lib/components/receive/ReceiveQRCode.svelte [hunk 3]
```diff
@@ -15,19 +27,36 @@
...
-		<QRCode value={address} />
+		<article
+			aria-label={replacePlaceholders($i18n.wallet.alt.qrcode_address, {
+				$token: symbol ?? ''
+			})}
+		>
+			<QRCode value={address}>
+				<svelte:fragment slot="logo">
+					{#if nonNullish(icon)}
+						<div class="flex items-center justify-center rounded-lg bg-white p-2">
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/receive/ReceiveQRCode.svelte:37]: Ah understood! thank you for the explanation

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**